client/my-sites/earn/components/add-edit-plan-modal/index.tsx

```diff
@@ -94,7 +94,7 @@ const RecurringPaymentsPlanAddEditModal = ( {
 	const currencyList = Object.keys( connectedAccountMinimumCurrency );
 
 	const defaultCurrency = useMemo( () => {
-		const flatCurrencyList = currencyList.map( ( e ) => e.toUpperCase() );
+		const flatCurrencyList = currencyList;
```

Comment:
Do we really need this, then?

Unecessary variable allocation make the code longer for no reason and can be confusing when working again on the code...

 Simply using `currencyList` here would be fine...

Reply:
I've removed this and updated where `flatCurrency` was used.